tests/Microsoft.Identity.Web.Test.Common/UserInformationFieldIds.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Identity.Lab.Api;
```

Comment:
This should ideally come from the NuGet package. can we add it here? thanks.

Reply:
It isnt technically supposed to be in there since it is related to Ui Automation testing rather than the acquisition of lab credentials but I will put it in there for now. Will need to consider refactoring that functionality in the future.